guides/how_to_use_2D_velocity_estimate.ipynb

```diff
@@ -1,4 +1,494 @@
 {
```

Comment:
I tried to open the notebook separately and got the following error: 
Invalid Notebook
The Notebook Does Not Appear to Be Valid JSON
Using nbformat v5.8.0 and nbconvert v7.3.1

Does anyone else have the same issue?

Reply:
Fixed